_posts/2023-05-11-Discontinuing-custom-Logstash-distribution.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Discontinuing the custom Logstash distribution"
+authors:
+- dlv
+date: 2023-05-11 12:30:00 -0500
+---
+
+One of the core [OpenSearch project principles](https://opensearch.org/about.html) is to be used everywhere. 
+Many users use [Logstash](https://www.elastic.co/logstash/) - a data processing pipeline - with OpenSearch. 
+To help OpenSearch be used everywhere, we provide two Logstash plugins to allow OpenSearch users to ingest data to and from OpenSearch using Logstash: [logstash-output-plugin](https://github.com/opensearch-project/logstash-output-opensearch) and [logstash-input-plugin](https://github.com/opensearch-project/logstash-input-opensearch). 
+We have also provided a custom distribution of Logstash with these plugins embedded. 
+We believe that OpenSearch users will continue to benefit from our plugins using only the Apache-licensed Logstash distribution from Elastic and plan to discontinue the custom Logstash distribution by October 2023.
+
+## Reasons to discontinue
+
+Many users are downloading the custom OpenSearch distribution of Logstash. 
+So it does provide some convenience and value to OpenSearch users. 
+However, we find some compelling reasons to discontinue the distribution.
+
+One of the reasons is to help users get updates quickly. 
+Because the OpenSearch distribution is based on the Apache-licensed distribution provided by Elastic, users cannot get the updates immediately after they are released. 
```

Comment:
This sentence confuses me. On line 13 we tell readers that our two plugins "use the Apache-licensed Logstash distribution from Elastic." But here it says the [custom?] distribution is based on the same "Apache-licensed distribution provided by Elastic." What exactly is the key difference between the plugins and the custom distribution that gives the plugins quicker updates?
Also, is it OK to start this sentence: "Because the custom OpenSearch distribution is based on ..." Having the word "custom" helps me distinguish between the distribution and the plugins.

Reply:
I'm not quite sure the best way to put this either. I pushed a change with another attempt.

I reworded to:

> Because the custom distribution is based on the Apache-licensed distribution provided by Elastic, users cannot get the updates immediately after Elastic releases them. 
Instead, they have to wait for the OpenSearch project to repackage the updates.


Thoughts?